README.md

```diff
@@ -159,6 +159,68 @@ As Android will not create our main activity before launching the app, we need t
 
 ```
 
+#### Deep Linking
```

Comment:
-> Handle deep Linking on android

(it's automatic on iOS)

Reply:
So, i added both sections!

Reply:
Yeah, it's not automatic in the sense that you have nothing to do to handle deep links on iOS 😅It's just that you have nothing more to do because of this package